src/core-sdk-tasks/CopyBlobsToLatest.cs

```diff
@@ -64,6 +64,11 @@ private AzurePublisher AzurePublisherTool
 
         public override bool Execute()
         {
+            if (!(Channel.Equals("master") || Channel.StartsWith("release")))
```

Comment:
Worth including Channel.Equals("main") in this change or do that separately when it comes time to move away from the master terminology?